### PR TITLE
Update ICRC-27 based on feedback

### PR DESCRIPTION
**Pull request by sea-snake**
After the last ICRC-27 meeting there was some feedback:

- Return all ICRC-1 accounts at once
- Clearify the standard in relation to ICRC-31 + ledger index canister
- How to trigger re-selecting accounts instead of returning previously selected

In this PR I've tried to update the ICRC-27 to address above points.

~~Regarding interactivity, I've opted for a required method argument over having separate methods for the following reasons:~~

- ~~With two methods, you would need to ask permission for both~~
- ~~With two methods, you would need to unnecessarily implement the same method signature twice~~
- ~~Some signers might not support interactive account selection, this is more clear with a single method~~
- ~~For the relying party the single method would behave same as long the signer supports the non interactive argument as required~~

Edit regarding interactivity, I've opted to make the get accounts method always interactive (prompt user to select accounts) as discussed in https://github.com/dfinity/wg-identity-authentication/issues/131.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,23 @@
...
-| [CAIP-2](https://github.com/icvc/icp-namespace/pull/1)<br>[CAIP-10](https://github.com/icvc/icp-namespace/pull/2) | CAIP Specifications                                                                                                                                               | [Quint Daenen](https://github.com/q-uint)                         | [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31) | [Issue 25](https://github.com/dfinity/wg-identity-authentication/issues/25), [CAIP Specifications](https://github.com/ChainAgnostic/CAIPs) |
+| [CAIP-2](https://github.com/icvc/icp-namespace/pull/1)<br>[CAIP-10](https://github.com/icvc/icp-namespace/pull/2) | CAIP Specifications
```
Comment (frederikrothenberger) [inline README.md:23]: You should probably not remove this. At least not without talking to Quint first. 😉

File: topics/icrc_27_get_icrc_1_accounts.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
+# ICRC-27: Get ICRC-1 Accounts
```
Comment (frederikrothenberger) [inline topics/icrc_27_get_icrc_1_accounts.md:1]: This PR only adds a file. You should also delete the other existing file `icrc_27_get_icrc_1_subbaccounts.md`.

**Review by frederikrothenberger — COMMENTED** [review]
Very nice work, only some clean-up left.

File: topics/icrc_27_get_icrc_1_accounts.md (hunk 2 above)
Comment (sea-snake) [reply topics/icrc_27_get_icrc_1_accounts.md:1]: Fixed, PR now correctly shows file as renamed.

File: README.md (hunk 1 above)
Comment (sea-snake) [reply README.md]: Fixed also double checked that nothing was removed and correctly updated the table column alignment

**Pull request merged**